library/src/main/java/uk/co/deanwild/materialshowcaseview/PrefsManager.java

```diff
@@ -47,7 +47,7 @@ int getSequenceStatus() {
 
     void setSequenceStatus(int status) {
         SharedPreferences internal = context.getSharedPreferences(PREFS_NAME, Context.MODE_PRIVATE);
-        internal.edit().putInt(STATUS + showcaseID, status).apply();
+        internal.edit().putInt(STATUS + showcaseID, status).commit();
```

Comment:
I should mention that these were changed because .apply() didn't show up until API 9.